ClarifaiTagger.lrdevplugin/ClarifaiAPI.lua

```diff
@@ -1,68 +1,56 @@
 local LrHttp = import 'LrHttp'
 local LrLogger = import 'LrLogger'
 local LrPathUtils = import 'LrPathUtils'
+local LrStringUtils = import 'LrStringUtils'
+local LrFileUtils = import 'LrFileUtils'
 local JSON = require 'JSON'
 local prefs = import 'LrPrefs'.prefsForPlugin(_PLUGIN.id)
 
 local logger = LrLogger('ClarifaiAPI')
 logger:enable('print')
 
 
-local tagAPIURL   = 'https://api.clarifai.com/v1/tag/'
-local tokenAPIURL = 'https://api.clarifai.com/v1/token/'
+local tagAPIURL   = 'https://api.clarifai.com/v2/models/aaa03c23b3724a16a56b629203edc62c/outputs'
```

Comment:
this id in the url doesnt look right. sure it belongs here?

Reply:
The id in the URL is the image recognition model ID for Carifai's "general" recognition model and not any sort of authentication id. See the model ID at this link

https://clarifai.com/models/general-image-recognition-model/aaa03c23b3724a16a56b629203edc62c

I'm guessing that's what you're asking.

Reply:
Yes, great. Thank you!
Now someone has to merge it :)
Will use your revision in the meantime.

Reply:
Tested it. Working great! (no crashes ;))